core/test_utils.go

```diff
@@ -14,15 +14,16 @@ import (
 )
 
 func CreateAndStartPeer(conf InitConfig, wait bool) (*Textile, error) {
-	_ = os.RemoveAll(conf.RepoPath)
-
 	conf.Account = keypair.Random()
+
+	_ = os.RemoveAll(conf.RepoPath())
```

Comment:
repo path is derived from the account, so we need to move this until after the account is created